lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
This file is pretty complicated, and ultimately makes the maintenance of this repo much harder (at least it adds significantly to the cognitive overhead required to fix/change things).  Maybe we should start out with the other changes, and circle back around to this one if it is absolutely critical...